### PR TITLE
feature: add CriEnabled field in api and show this in cli

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

### Ⅰ. Describe what this PR did
In daemon's API side, we need to provide a user-friendly way for user to check if the pouchd is opening for CRI part.

Otherwise, user can only use `ps aux | grep pouchd | grep cri-enable` to check, which is quite inconvenient. In addition, if a pouch client can only access pouchd remotely, no info can be gotten with this way. So, I think we need to add CriEnabled field in api and show this in cli.

### Ⅱ. Does this pull request fix one issue?

fixes https://github.com/alibaba/pouch/issues/1908

### Ⅲ. Describe how you did it
1. Add in swagger.yml
2. return in daemon side;
3. show in cli side

### Ⅳ. Describe how to verify it

execute `pouch info`, you will see `CriEnabled: false` if no cri-enabled, otherwise, true.

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

File: test/z_cli_daemon_test.go [hunk 1]
```diff
@@ -389,6 +389,20 @@ func (suite *PouchDaemonSuite) TestDaemonDefaultRegistry(c *check.C) {
 	defer dcfg.KillDaemon()
 }
 
+// TestDaemonCriEnbaled tests enabling cri part in pouchd.
+func (suite *PouchDaemonSuite) TestDaemonCriEnbaled(c *check.C) {
+	dcfg, err := StartDefaultDaemonDebug(
+		"--enable-cri")
+	c.Assert(err, check.IsNil)
+
+	// Check pull image with default registry using the registry specified in daemon.
```
Comment (YaoZengzeng) [inline test/z_cli_daemon_test.go]: This comment is not appropriate 😄

**Comment by YaoZengzeng** [comment]
LGTM, except the minor issue mentioned above :)

**Comment by allencloud** [comment]
Thanks for your review. @YaoZengzeng 
I have removed the comment. Until it passes the CI, I think we could merge this.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**